automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -60,6 +64,36 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
       }
     }
 
+    "should execute Hail with correct permissions on a cluster with preemptible workers" in withWebDriver { implicit driver =>
+      withNewBillingProject { project =>
+        Orchestration.billing.addUserToBillingProject(project.value, ronEmail, Orchestration.billing.BillingProjectRole.User)(hermioneAuthToken)
+
+        withNewGoogleBucket(project) { bucket =>
+          val srcPath = parseGcsPath("gs://genomics-public-data/1000-genomes/vcf/ALL.chr20.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf").right.get
+          val destPath = GcsPath(bucket, GcsObjectName("chr20.vcf"))
+          googleStorageDAO.copyObject(srcPath.bucketName, srcPath.objectName, destPath.bucketName, destPath.objectName)
```

Comment:
hard-coded srcPath worries me. it is a common reason for test to break.

Reply:
It's a very large file.  I don't see a good alternative to keeping it in a bucket.